lib/models/executionsStore.js

```diff
@@ -42,19 +42,27 @@ module.exports = {
                 myutils.logErrorIf(err);
```

Comment:
CHANGES_NEXT_RELEASE entry missing

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/340/commits/13388f37c7ab09c1b5f0d156d632aa7cd8df5ca0